### PR TITLE
New version: RustDesk.RustDesk version 1.2.0

### PR DESCRIPTION
**Pull request by sitiom**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111231&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/r/RustDesk/RustDesk/1.2.0/RustDesk.RustDesk.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Created with Komac v1.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RustDesk.RustDesk
+PackageVersion: 1.2.0
+PackageLocale: en-US
+Publisher: RustDesk
+PublisherUrl: https://rustdesk.com
+PublisherSupportUrl: https://github.com/rustdesk/rustdesk/issues
+PrivacyUrl: https://rustdesk.com/privacy
```
Comment (CoolPlayLin) [inline manifests/r/RustDesk/RustDesk/1.2.0/RustDesk.RustDesk.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://rustdesk.com/privacy.html
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
image

**Comment by sitiom** [comment]
@stephengillie Any idea what went wrong here?

**Comment by stephengillie** [comment]
Manual Validation failed with:
> ##[error] Installer https://github.com/rustdesk/rustdesk/releases/download/1.2.0/rustdesk-1.2.0-x86_64.exe
Scan request id: '2091bb05-88d9-4512-97f8-20745062349e'
Error information: BlockingDetectionFound
##[error] Result: Failed

**Review by stephengillie — APPROVED** [review]

**Pull request merged**